README.rst

```diff
@@ -135,6 +135,10 @@ not officially supported, and will not receive bugfixes or new features.
         For packages not found in the local index, this URL will be used to
         redirect to (default: https://pypi.org/simple/).
 
+      --health-endpoint HEALTH_ENDPOINT
+        Configure a custom liveness endpoint. It always returns 200 Ok if 
+        the service is up. Otherwise it means that the service is dead.
```

Comment:
```suggestion
        Configure a custom liveness endpoint. It always returns 200 Ok if  
        the service is up. Otherwise, it means that the service is not responsive.
```

Just this tiny detail for consistency, otherwise it's great to go! :) Sorry for taking such a long time for the review 😅